tests/bson/bson-encode-002.phpt

```diff
@@ -67,7 +67,7 @@ $bson = fromPHP(array('embed' => new NumericArray));
 echo toJSON($bson), "\n";
 echo "Encoded BSON:\n";
 hex_dump($bson);
-$value = toPHP($bson, array("document" => 'NumericArray'));
+$value = toPHP($bson, array("array" => 'NumericArray'));
```

Comment:
The original test was written before allowing Serializables to encode as a BSON array. The original intention of this test is just to round-trip the data from NumericArray to BSON and back to NumericArray. Since the value does get stored as a BSON array, the type map needed an update.